packages/hardhat-plugin/src/Unlock.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable class-methods-use-this, import/no-cycle */
+import networks from '@unlock-protocol/networks'
```

Comment:
Ok so that's an interesting challenge because I think we should avoid publishing this package. (it contains API keys for Alchemy and others). But we need to expose the addresses for the contracts...
So I think the best approach would be to create a specific network config file here with the stuff the plugin needs _but_ ask the user to put their own RPC providers. wdyt?

Reply:
I added a build task `yarn networks` to pre-package the networks files